### PR TITLE
Market decentralized - additional logs

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**
- logs improvements
- increase broadcast mean interval

### CODE REVIEW

**Review by tworec — DISMISSED** [review]
lgtm

**Review by prekucki — APPROVED** [review]

**Review by jiivan — APPROVED** [review]

File: core/market/decentralized/src/config.rs [hunk 1]
```diff
@@ -23,10 +23,10 @@ pub struct SubscriptionConfig {
...
     fn default() -> Self {
         DiscoveryConfig {
-            max_bcasted_offers: 50,
-            max_bcasted_unsubscribes: 50,
-            mean_cyclic_bcast_interval: Duration::from_secs(3),
-            mean_cyclic_unsubscribes_interval: Duration::from_secs(3),
+            max_bcasted_offers: 200,
+            max_bcasted_unsubscribes: 200,
+            mean_cyclic_bcast_interval: Duration::from_secs(60),
+            mean_cyclic_unsubscribes_interval: Duration::from_secs(60),
```
Comment (tworec) [inline core/market/decentralized/src/config.rs:29]: 👍

File: Cargo.toml [hunk 2]
```diff
@@ -104,6 +104,7 @@ members = [
     "utils/compile-time-utils",
     "utils/path",
     "utils/process",
+    "utils/std-utils",
```
Comment (tworec) [inline Cargo.toml:107]: a project with such name means 'trash bin for various code that not belong anywhere' usually ;)

File: core/market/decentralized/src/matcher/handlers.rs [hunk 3]
```diff
@@ -103,7 +103,9 @@ pub(super) async fn receive_remote_offer_unsubscribes(
                     .map(|_| offer_id.clone())
                     .map_err(|e| match e {
                         // We don't want to warn about normal situations.
-                        ModifyOfferError::Unsubscribed(..) | ModifyOfferError::Expired(..) => e,
+                        ModifyOfferError::Unsubscribed(..)
+                        | ModifyOfferError::Expired(..)
+                        | ModifyOfferError::NotFound(..) => e,
```
Comment (tworec) [inline core/market/decentralized/src/matcher/handlers.rs:108]: 👍

File: core/net/src/service.rs [hunk 4]
```diff
@@ -71,7 +71,7 @@ pub async fn bind_remote(default_node_id: NodeId, nodes: Vec<NodeId>) -> std::io
             let endpoints = bcast.resolve(&topic);
             let msg: Rc<[u8]> = msg.into();
             Arbiter::spawn(async move {
-                log::debug!("Received broadcast to topic {} from [{}].", &topic, &caller);
+                log::trace!("Received broadcast to topic {} from [{}].", &topic, &caller);
```
Comment (tworec) [inline core/net/src/service.rs:74]: 👍

**Pull request merged**